src/ontology/config/prefixes.csv

```diff
@@ -227,12 +227,27 @@ ICD10CM,http://purl.bioontology.org/ontology/ICD10CM/
 ICD10CM2,https://icd.codes/icd10cm/
 ICD10WHO,https://icd.who.int/browse10/2019/en#/
 ICD10WHO2010,http://apps.who.int/classifications/icd10/browse/2010/en#/
-OMIMPS,https://www.omim.org/phenotypicSeries/PS
+OMIMPS,https://omim.org/phenotypicSeries/PS
```

Comment:
## `prefixes.csv` updates
That will be supplanted eventually by dynamic creation of `prefixes.csv` but these changes seem necessary for now.

Harshad and I should have thought to check the status of this sooner.